django/core/management/commands/makemessages.py

```diff
@@ -188,7 +188,9 @@ def write_pot_file(potfile, msgs):
                 header_read = True
             lines.append(line)
     msgs = '\n'.join(lines)
-    with open(potfile, 'a', encoding='utf-8') as fp:
+    # Forcing POT files newline because of a gettext bug:
```

Comment:
suggested comment: `# Force newlines of POT files to '\n' to work around https://savannah.gnu.org/bugs/index.php?52395`